Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c

```diff
@@ -183,7 +183,7 @@ GetCpuFamily (
   VOID
   )
 {
-  EFI_CPUID_REGISTER Cpuid;
+  EFI_CPUID_REGISTER Cpuid = { 0, 0, 0, 0 };
```

Comment:
the Cpuid registers get initial value in AsmCpuid routine.
wonder if scanning can analyze assembly code.

Reply:
Will remove this commit id.